### PR TITLE
Close headless signer if no connection from terminal after 5 s

### PR DESCRIPTION
**Pull request by Ation**

### CODE REVIEW

File: BlockSettleSigner/main.cpp [hunk 1]
```diff
@@ -316,14 +319,22 @@ static int QMLApp(int argc, char **argv
...
 
       switch (settings->runMode()) {
       case bs::signer::ui::RunMode::fullgui:
          engine.load(QUrl(QStringLiteral("qrc:/qml/main.qml")));
          break;
       case bs::signer::ui::RunMode::litegui:
          engine.load(QUrl(QStringLiteral("qrc:/qml/mainLite.qml")));
+
+         terminalConnectionTimer.setSingleShot(true);
+         //http://185.213.153.35:8081/browse/BST-2786
```
Comment (goatpig) [inline BlockSettleSigner/main.cpp:331]: I'm guessing anyone can access this IP, scrub that out from the comment, just leave teh BST code

**Comment by goatpig** [comment]
This kills the signer within 5seconds of starting the terminal, while the terminal is live.

**Comment by Ation** [comment]
I was not able to reproduce, but that could happen only if terminal connects to signer before signer "ui" ready to receive such notifications. I'll update code to avoid such situation.

**Comment by Ation** [comment]
Double checked. Looks correct. Can you force rebuild on a project and check one more time?

**Comment by goatpig** [comment]
Will do

**Comment by goatpig** [comment]
Getting the same error.

**Review by goatpig — APPROVED** [review]

**Pull request merged**